webpack/scenes/ChangeContentSource/components/cs_form.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+  ActionGroup,
+  Button,
+  Form,
+  Grid,
+  GridItem,
+} from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import PropTypes from 'prop-types';
+
+import { formField } from '../helpers';
+
+const CsForm = ({
```

Comment:
I'm not very keen on abbreviations like this. `ContentSourceForm` seems much nicer